index.js

```diff
@@ -25,65 +26,74 @@ function getRandomNumber(min, max) {
 function checkGuess() {
   // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
+  attempts += 1;
 
   hideAllMessages();
 
   if (guess === targetNumber) {
     numberOfGuessesMessage.style.display = '';
     numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
-
     correctMessage.style.display = '';
-
     submitButton.disabled = true;
     guessInput.disabled = true;
+    resetButton.style.display = '';
+
   }
 
   if (guess !== targetNumber) {
     if (guess < targetNumber) {
       tooLowMessage.style.display = '';
     } else {
-      tooLowMessage.style.display = '';
+      tooHighMessage.style.display = '';
     }
 
     const remainingAttempts = maxNumberOfAttempts - attempts;
 
     numberOfGuessesMessage.style.display = '';
     numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
+    resetButton.style.display = '';
   }
 
-  if (attempts ==== maxNumberOfAttempts) {
+  if(guess > maxNumberOfAttempts){
+    alert `Remember: is 1 to 99`;
+    resetButton.style.display = '';
+  }
+
+  if (attempts === maxNumberOfAttempts) {
     submitButton.disabled = true;
     guessInput.disabled = true;
+    maxGuessesMessage.style.display = '';
+    maxGuessesMessage.innerHTML = `0 guesses remaining`;
```

Comment:
The HTML already has the error messages declared, you don't need to change the HTML like this. You just need to make sure that existing message becomes visible. 

The line above this one already does this so you're duplicating the message.